veinminer-bukkit/src/main/java/wtf/choco/veinminer/VeinMinerPlugin.java

```diff
@@ -26,16 +26,7 @@
 import org.jetbrains.annotations.UnmodifiableView;
 
 import wtf.choco.network.bukkit.BukkitProtocolConfiguration;
-import wtf.choco.veinminer.anticheat.AntiCheatHook;
-import wtf.choco.veinminer.anticheat.AntiCheatHookAAC;
-import wtf.choco.veinminer.anticheat.AntiCheatHookAntiAura;
-import wtf.choco.veinminer.anticheat.AntiCheatHookGrim;
-import wtf.choco.veinminer.anticheat.AntiCheatHookLightAntiCheat;
-import wtf.choco.veinminer.anticheat.AntiCheatHookMatrix;
-import wtf.choco.veinminer.anticheat.AntiCheatHookNCP;
-import wtf.choco.veinminer.anticheat.AntiCheatHookNegativity;
-import wtf.choco.veinminer.anticheat.AntiCheatHookSpartan;
-import wtf.choco.veinminer.anticheat.AntiCheatHookThemis;
+import wtf.choco.veinminer.anticheat.*;
```

Comment:
All imports should be kept, please :)

Reply:
I understand the other changes, but just for curiosity. Why are you against using .*?